web/src/pill_typeahead.ts

```diff
@@ -86,10 +87,12 @@ export function set_up_stream(
         type: "contenteditable",
     };
     opts.help_on_empty_strings ||= false;
```

Comment:
Should `opts.help_on_empty_strings` also use nullish operator for uniformity?

Reply:
Yeah, nullish operator is generally preferred, to be more clear that we only fall back to value when it's undefined.

Reply:
Done.

Reply:
Please put that in a different commit though

Reply:
My bad I misunderstood your comment:

> I think it would make more sense to put this file's changes in the last commit


I have fixed it now.

Reply:
Oh I see. Yeah I meant that file's changes _within that commit_, not also this unrelated change you hadn't made yet. Thanks for moving it!